src/MailTrackerController.php

```diff
@@ -92,7 +87,9 @@ protected function linkClicked($url, $hash)
             }
         }
 
-        if ( ! $tracker || empty($tracker->domains_in_context) || ! in_array($url_host, $tracker->domains_in_context) ){
+        // Only check the domains in context if we are storing the content.. If we aren't then we will have to rely on the ::signedRoute to protect us from unwanted links
+        // TODO: This can be removed eventually when we enforce the use of signed routes for all links with no grace period
+        if (config('mail-tracker.log-content', true) && (!$tracker || empty($tracker->domains_in_context) || !in_array($url_host, $tracker->domains_in_context))) {
```

Comment:
I'm still not sure on this long term, once signedRoutes are in use then we don't need this at all. 

I would potentially check the start date/grace period and if it's after that then just ignore this check